src/features/Treege/Forms/FlowNodeForm.tsx

```
@@ -0,0 +1,68 @@
+    <form
+      id="flow-node-form"
+      onChange={handleSubmit}
+      onSubmit={(e) => {
+        e.preventDefault();
+        e.stopPropagation();
+      }}
+    >
+      <div className="grid gap-6">
+        <Field
+          name="label"
+          children={(field) => (
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)